Dev4Agriculture.ISO11783.ISOXML.Generation/ISOXMLNamingProvider.cs

```diff
@@ -58,7 +61,8 @@ public override string EnumMemberNameFromValue(string enumName, string value, Xm
         if (documentations.Count > 0)
         {
             var name = Regex.Replace(documentations[0].Text, @"\t|\n|\r|,|(\s+)", "");
-            return base.EnumMemberNameFromValue(enumName, name, xmlFacet);
+            var result =  base.EnumMemberNameFromValue(enumName, name, xmlFacet);
+            return _enumWordsToRemove.Aggregate(result, (current, word) => current.Replace(word, string.Empty));
```

Comment:
2 things here:
1.) I think it makes sense to add an _ to differentiate if brackets or no brackets. Please remove _ at the very end of the properties
2.) The First Letter within the Brackets is sometimes lowercase. Could you fix that as well?